### ISSUE TITLE
attach air pollution data to per-city indicators

### ISSUE THREAD

**Issue by gboeing** [healthysustainablecities/global-indicators#94]
@carlhiggs on the call today I was reminded that we still need to attach the city-level air pollution data to our indicators output. The relevant variables would be:

  - E_EC2E_T15
  - E_EC2O_T15
  - E_EPM2_T15
  - E_CPM2_T14

You have a local copy of the urban centers database, right? If so would you mind quickly attaching those columns from it to our indicators output in the code?

**Comment by carlhiggs** [comment]
Hi @gboeing,

Sure happy to do this - perhaps the easiest way might be to retrieve it for
those cities where it's available (all but Vic, I think, which I guess
could have a null instead) as a summary statistic in the urban study region
layer in the pre process stage, so it's there in the input gpkg to be
excerpted later when doing the aggregation stage?

Anyway, I agree it shouldn't take long to modify code to include this, as
well as the overall hex and region population and intersection densities,
which I think we're also requested last week.  Thanks for pointing to the
required variable names.

I'll look to making these updates for delivery early next week (while the
last update, with re parameterisation is pretty much done and uploaded. --
I just want to check the output a little more to make sure there aren't any
city oddities unaccounted for with changes I've made--, I've got to work on
the Bangkok project for a deliverable on Friday).

Hope that works,

Carl

**Comment by gboeing** [comment]
Sounds great. Thanks!

**Comment by carlhiggs** [comment]
Hi @gboeing , I've just pushed up changes to the reparameterise_1k branch which includes the urban covariates when producing the final outputs.  I made a few additional simplifications along the way, and I'll check the output properly on Monday - but just wanted to let you know that this seems to be working now, pending final verification.

After having a quick look at the output, and following a discussion thread in our meeting the other day, one thing that I think would be really good before we sign off on everything if someone could review the method used to estimate area level population using the GHS data using zonal stats.   But anyway we can talk about that later; I'll update all next week after I've actually checked out the outputs.  In the meantime, they should be uploading to an data/output/October final folder on Cloudstor.   Enjoy the weekend :)
